elastic/kibana
# [TSVB] Documents the new index pattern mode

## Summary

Closes https://github.com/elastic/kibana/issues/102323

In our effort to push more users to use the index pattern mode I:
- Added a section for it on TSVB docs. You can see the preview here:
https://kibana_102880.docs-preview.app.elstc.co/guide/en/kibana/master/tsvb.html#tsvb-index-pattern-mode
- Added a callout to TSVB 
![image](https://user-images.githubusercontent.com/17003240/123604549-9b7b9080-d803-11eb-8f6b-7700d3c1fba4.png)

Update: After discussing it with the design team, the callout appears only for old visualizations that use the string mode. We also give the users the ability to dismiss the notification.

### How to test it
- For the new visualizations (and the sample data as we have updated them to use the index pattern mode) the callout should be hidden
- Import this https://gist.github.com/stratoula/c936605bed52b02b0278ba872096598f (7.12.1 visualization). Callout should be visible and not appear again after it is dismissed.

### Checklist
